synapse/config/workers.py

```diff
@@ -104,6 +104,23 @@ def netloc(self) -> str:
         return f"{self.host}:{self.port}"
 
 
+class InstanceUNIXLocationConfig(ConfigModel):
```

Comment:
We spell `Unix` without the capitalization elsewhere. Probably align with that

https://github.com/matrix-org/synapse/blob/ba97b39881e296f4775b8f6dd18edb98a3dc733f/stubs/txredisapi.pyi#L66

https://github.com/matrix-org/synapse/blob/ba97b39881e296f4775b8f6dd18edb98a3dc733f/synapse/config/server.py#L240

Reply:
As long as pairs of functions match up consistently, I'm not married to either way. Which is funny because the two `ListenerConfig`s are something I wrote 🤦‍♂️ I'll look into matching these current two to a matching style consistent with the majority in Synapse. I suppose at the time I was thinking that Unix was a word/name whereas TCP was an acronym.